global_logger.py

```diff
@@ -1,20 +1,32 @@
 import logging.handlers
 import logging.handlers
 import os
+import pprint
 import time
 from enum import auto, Enum
 from functools import wraps
 from typing import Optional, Union, Callable, Collection
+from util.const import ConstContainer
 
 # Set up logging level constants
 from util.strings import cslist
 
-CRITICAL = logging.CRITICAL  # 50
-ERROR = logging.ERROR  # 40
-WARNING = logging.WARNING  # 30
-INFO = logging.INFO  # 20
-VERBOSE = 15
-DEBUG = logging.DEBUG  # 10
+
+class LogLevels(ConstContainer):
+    CRITICAL = logging.CRITICAL  # 50
+    ERROR = logging.ERROR  # 40
+    WARNING = logging.WARNING  # 30
+    INFO = logging.INFO  # 20
+    VERBOSE = 15
+    DEBUG = logging.DEBUG  # 10
+
+
+CRITICAL = LogLevels.CRITICAL  # 50
+ERROR = LogLevels.ERROR  # 40
+WARNING = LogLevels.WARNING  # 30
+INFO = LogLevels.INFO  # 20
+VERBOSE = LogLevels.VERBOSE # 15
+DEBUG = LogLevels.DEBUG  # 10
```

Comment:
I created the container so I could easily loop through all levels in my unit tests. You can still access them as constants directly in the module just like before.